File: RxJsoupNetWorkLibrary/src/main/java/io/reactivex/jsoup/network/bus/RxBusObserver.java
package io.reactivex.jsoup.network.bus;

import io.reactivex.annotations.NonNull;
import io.reactivex.observers.DisposableObserver;

/**
 * by y on 2017/5/22
 */

class RxBusObserver<T> extends DisposableObserver<T> {
    @Override
    public void onNext(@NonNull T t) {

    }

    @Override
    public void onError(@NonNull Throwable e) {

    }

    @Override
    public void onComplete() {

    }
}

File: RxNetWorkLibrary/src/main/java/io/reactivex/network/bus/RxBusObserver.kt
package io.reactivex.network.bus

import io.reactivex.annotations.NonNull
import io.reactivex.observers.DisposableObserver

/**
 * by y on 2017/5/22
 */

internal open class RxBusObserver<T> : DisposableObserver<T>() {
    override fun onNext(@NonNull t: T) {

    }

    override fun onError(@NonNull e: Throwable) {

    }

    override fun onComplete() {

    }
}
